codebase/src/java/disco/org/openlvc/disco/configuration/DisConfiguration.java

```diff
@@ -148,6 +157,15 @@ public void setAppId( int appId )
 		this.appId = appId;
 	}
 	
+	public Map<PduType,Supplier<PDU>> getPduSuppliers()
```

Comment:
Given that this is where user apps will be doing this PDUSupplier stuff, should probably have some decent javadoc on these methods